### PR TITLE
Homework1: Ivan Veremeichik

### PR DESCRIPTION
**Pull request by NorgeMan**
### PR checklist
Please make sure you've passed PR checklist
- [x] my PR passes CI checks (flake8 and pycodestyle).
- [x] my PR named according to guidelines.
- [x] my PR contains complete tasks descriptions for all tasks.
- [x] my PR made from the fork feature branch to appropriate branch.
- [x] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore.

### CODE REVIEW

File: src/homework1.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    print('This generation has {0} babies'.format(babies))
+    parents, babies = (babies, parents + babies)
+
+
+# 5 lines: Functions
+def greet(name):
+    print('Hello', name)
+
+
+greet('Jack')
```
Comment (belovmd) [inline src/homework1.py:34]: you have 2 lines between a function definition and it's call and you have only one line between It and next task. I suggest such pattern:

```python
   pass  # last line of previous task
    
 
   # next task docstring
   pass  # my next task first line  
```

File: src/homework1.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+my_purchase = {
+    'apple': 1,
+    'banana': 6}
+grocery_bill = sum(prices[fruit] * my_purchase[fruit]
+                   for fruit in my_purchase)
+print('I owe the grocer $%.2f' % grocery_bill)
+
+# 8 lines: Command line arguments, exception handling
+# This program adds up integers that have been passed as arguments in the command line
+
```
Comment (belovmd) [inline src/homework1.py]: usually, you have 0 lines between docstring and code and here you have 1 empty line between it. you code should be consistent.

File: src/homework1.py [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+try:
+    total = sum(int(arg) for arg in sys.argv[1:])
+    print('sum =', total)
+except ValueError:
+    print('Please supply integer arguments')
+
+# 9 lines: Opening files
+# indent your Python code to put into an email
+# glob supports Unix style pathname extensions
+
```
Comment (belovmd) [inline src/homework1.py]: same issue

**Review by belovmd — CHANGES_REQUESTED** [review]

**Review by NorgeMan — COMMENTED** [review]
code was fixed after review

**Review by belovmd — APPROVED** [review]
1

**Pull request closed without merging**